docusaurus.config.js

```diff
@@ -127,10 +127,10 @@ module.exports = {
               rel: null,
             },
             {
-              href: '/',
-              label: 'JWZ (coming soon)',
+              href: 'https://jwz.polygonid.me',
+              label: 'JWZ',
               target: '_blank',
-              className: "disabled",
+              rel: null,
```

Comment:
rel should have the value "noopener noreferrer"

Reply:
I've removed all the `target` and `rel` attributes from the links since docusaurus already adds them.